contentful/content_type.py

```diff
@@ -20,20 +20,8 @@ class ContentType(Resource):
     API Reference: https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/content-types
     """
 
-    def __init__(
-            self,
-            item,
-            default_locale='en-US',
-            includes=None,
-            localized=False,
-            depth=0):
-        super(ContentType, self).__init__(
-            item,
-            default_locale,
-            includes,
-            localized,
-            depth
-        )
+    def __init__(self, item, **kwargs):
```

Comment:
These calls are equivalent, all that's declared like `something=default_value` is enclosed within **kwargs. The only reason to unpack it would be if it would be required within the method call. In this case only `item` is relevant, therefore making the constructor having an explicit declaration of everything irrelevant.